share/picongpu/examples/Bunch/include/picongpu/param/fieldBackground.param

```diff
@@ -20,20 +20,18 @@
 #pragma once
 
 /** Load pre-defined templates */
-#ifndef TWTSFAST
-#    include "picongpu/fields/background/templates/TWTS/TWTS.hpp"
-#endif
-
-#ifdef TWTSFAST
+#if PARAM_TWTSFAST == 1
 #    include "picongpu/fields/background/templates/twtsfast/twtsfast.hpp"
+#else
+#    include "picongpu/fields/background/templates/TWTS/TWTS.hpp"
 #endif
 
-#ifndef PARAM_INCLUDE_FIELDBACKGROUND
+ifndef PARAM_INCLUDE_FIELDBACKGROUND
```

Comment:
BUG: you removed the `#`

Reply:
You must have looked at a previous commit. This is already fixed.